manifests/1.1.0/opensearch-dashboards-1.1.0.yml

```diff
@@ -24,9 +24,9 @@ components:
   # - name: indexManagementDashboards
   #   repository: https://github.com/opensearch-project/index-management-dashboards-plugin
   #   ref: main
-  # - name: anomalyDetectionDashboards
-  #   repository: https://github.com/opensearch-project/anomaly-detection-dashboards-plugin
-  #   ref: main
+  - name: anomalyDetectionDashboards
+    repository: https://github.com/opensearch-project/anomaly-detection-dashboards-plugin
+    ref: main
```

Comment:
Do they have a 1.1 branch yet?

Reply:
Same question, they should be about to release 1.1, is expect a branch and for main to be 1.2.